models/like.js

```diff
@@ -0,0 +1,69 @@
+'use strict';
+
+module.exports = (sequelize, DataTypes) => {
+  var like = sequelize.define('like', {
+    user_id: DataTypes.INTEGER,
+    model_name: DataTypes.STRING,
+    model_id: DataTypes.STRING
+  }, 
+  {
+    underscored: true
+  },
+  {
+    classMethods: {
+      associate: function(models) {
+        // associations can be defined here
+      },
+      get_like_by_user: function(post_id, user_id) {
+        return Like.findOne({
+          where: {
+            user_id: user_id,
+            model_id: post_id
+          }
+        })
+        .then(like => {
+          console.log("Successfully found liked post: ", like);
+          return like;
+        })
+        .catch(err => {
+          console.log("Failed to find liked post");
+          return {error: err, status: 500};
```

Comment:
yes this is the idea - we're just concerned about the data here. the status code is also http though, we should move that up a level out of this file too